packages/ui-components/src/components/InputGroup/InputGroup.component.tsx

```diff
@@ -68,11 +77,10 @@ export const InputGroup: React.FC<InputGroupProps> = ({
     cloneElementWithInheritedProps(child, variant, disabled)
   )
 
+  const inputGroupClassName = getClassNames("juno-input-group", variant, disabled, className)
+
   return (
-    <Stack
-      className={`juno-input-group juno-input-group-${variant} ${disabled ? "juno-input-group-disabled" : ""} ${className}`}
-      {...props}
-    >
+    <Stack className={inputGroupClassName} {...props}>
```

Comment:
See explanation above:
```suggestion
    <Stack className={`juno-input-group ${inputGroupClassName} ${className}`} {...props}>
```